eng/Publishing.props

```diff
@@ -1,5 +1,21 @@
 <Project>
   <PropertyGroup>
     <PublishingVersion>3</PublishingVersion>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishPackages</PublishDependsOnTargets>
   </PropertyGroup>
+
+  <ItemGroup>
+    <_PackagesToPublish Remove="@(_PackagesToPublish)" />
```

Comment:
Why do you need to remove from the existing item? Can't you just define a new item instead?

Reply:
Seems cleaner and follows existing model in `aspnetcore`.

Reply:
IMO removing from an existing item is not cleaner than defining a new one. Removing from an item can have unintended side effects while defining a new item with a unique name shouldn't have any side effects.

Reply:
Resolving with latest commit - https://github.com/NikolaMilosavljevic/fsharp/commit/10c6a33f3fd759dc67f8cae13235e1c50473f404

Reply:
Fixed with https://github.com/dotnet/fsharp/pull/16838/commits/10c6a33f3fd759dc67f8cae13235e1c50473f404